org.lflang.tests/src/org/lflang/tests/TestBase.java

```diff
@@ -696,11 +706,9 @@ private void validateAndRun(Set<LFTest> tests, Configurator configurator, TestLe
                     test.result = Result.TEST_PASS;
                 }
 
-            } catch (AssertionError e) {
-                // Do not report assertion errors. They are pretty printed
-                // during reporting.
-            } catch (Exception e) {
-                test.issues.append(e.getMessage());
+            } catch (TestExecutionException e) {
+                // This exception is thrown by any of the test steps above. We mainly use the exception
+                // to abort test execution. There is no need to report the error here.
```

Comment:
This is where the exception should be handled though, so that failures in any of the substeps are handled uniformly.

Reply:
I think that I have run into issues with the test framework swallowing exceptions before. If a message were printed here, then there would be less risk that the code will change slightly and then start swallowing exceptions again.